pkg/packagekit/updates.jsx

```diff
@@ -265,9 +265,19 @@ class UpdateItem extends React.Component {
 
         var type;
         if (info.severity === PK_INFO_ENUM_SECURITY) {
+            var classes = "pficon pficon-security";
+
+            // parse Red Hat security update classification from vendor_urls
+            var secSeverity = info.vendor_urls.find(value => value.startsWith("https://access.redhat.com/security/updates/classification/#"));
+            if (secSeverity) {
+                var s = secSeverity.slice(secSeverity.indexOf("#") + 1);
+                classes += " severity-" + s;
```

Comment:
Let's validate that `s` is one of "critical", "important" or "low" before using it as part of a class name, because paranoia.

Reply:
OK, can do; I didn't yet do that because of hardcoding severity levels, but see above. Once we do it for sorting, we can just as well do it for validation here.

Reply:
The levels are already hard-coded in the CSS, no?  Hard-coding them in two places instead of one is worse, of course, but a comment in the CSS could help people that want to add more.

I am just worried that we might put nonsense strings into the className property.